frontend/src/main/routes.js

```diff
@@ -42,6 +43,8 @@ export default props => (
         path='/task/:id/order/:order_id/status/:status'
         component={ TaskOrdersContainer }
       />
+      <Route path='/404' component={ FourOFour } />
+      <Route component={ FourOFour } />
```

Comment:
@ShubhamCanMakeCommit why we have two routes here?

Reply:
The first one is the route that is called when no task exists and the last route is the default route that would match if no other route matches. e.g. `gitpay.me/#/anything/random/here` .